hack/verify-toc.sh

```diff
@@ -47,6 +47,7 @@ echo "Checking table of contents are up to date..."
 find keps -name '*.md' \
     | grep -Fxvf hack/.notableofcontents \
     | xargs mdtoc --inplace --max-depth=5 --dryrun || (
-      echo "Table of content not up to date. If this failed silently and you are on mac, try 'brew install grep'"
+      echo "Table of content not up to date. Did you run 'make update-toc'?"
+      echo "If this failed silently and you are on mac, try 'brew install grep'"
```

Comment:
split this out as a separate PR?

Reply:
Thanks, fixed it. it got in by accident - I already opened another PR for that.